java/registry/src/main/java/io/opensaber/registry/controller/RegistryController.java

```diff
@@ -175,18 +175,22 @@
         Response response = new Response(Response.API_ID.CREATE, "OK", responseParams);
         Map<String, Object> result = new HashMap<>();
         String entityType = apiMessage.getRequest().getEntityType();
+        JsonNode rootNode = apiMessage.getRequest().getRequestMapNode();
 
         try {
+            Shard shard = shardManager.getShard(rootNode.get(entityType).get(shardManager.getShardProperty()));
+            registryHelper.addEntity(rootNode, apiMessage.getUserID());
```

Comment:
Can the registryHelper continue to return String?

Reply:
Changed the return type to String.